server/game/cards/13.4-BtRK/ShadowblackAssassin.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard');
+
+class ShadowblackAssassin extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card === this
+            },
+            target: {
+                type: 'select',

```

Comment:
The card text says 'choose' so this isn't needed. `type: 'select'` is only used for abilities that have implicit targeting For example, "place a token on a character" like Tears of Lys uses it because it doesn't say to choose the character.